cmd/asset-syncer/main.go

```diff
@@ -42,9 +42,14 @@ func init() {
 
 	for _, cmd := range cmds {
 		rootCmd.AddCommand(cmd)
+		cmd.Flags().String("database-type", "mongodb", "Database to use. Choice: mongodb, postgresql")
 		cmd.Flags().String("mongo-url", "localhost", "MongoDB URL (see https://godoc.org/github.com/globalsign/mgo#Dial for format)")
 		cmd.Flags().String("mongo-database", "charts", "MongoDB database")
 		cmd.Flags().String("mongo-user", "", "MongoDB user")
+		cmd.Flags().String("pg-host", "localhost", "PostgreSQL Hostname")
+		cmd.Flags().String("pg-port", "5432", "PostgreSQL Port")
+		cmd.Flags().String("pg-database", "assets", "PostgreSQL database")
+		cmd.Flags().String("pg-user", "", "PostgreSQL user")
```

Comment:
As above, can we pls parse these in the main.go and create the implementation of your interface here, or fatal here?